openassessment/assessment/tasks.py

```diff
@@ -1,7 +0,0 @@
-"""
-Celery looks for tasks in this module,
-so import the tasks we want the workers to implement.
-"""
-# pylint:disable=W0611
-from .worker.training import train_classifiers, reschedule_training_tasks
-from .worker.grading import grade_essay, reschedule_grading_tasks
```

Comment:
Everything under worker has been removed - the idea was to "train" your AI classifiers with a given data set ahead of time, then "grade" new essays as a separate phase. We don't need any references to either of these operations.